_layouts/integration.html

```diff
@@ -73,6 +86,36 @@ <h2>API support</h2>
   </details>
 </section>
 
+{% if page.quality_estimation_integrations.size > 0 %}
```

Comment:
I'd probably just leave that section if it has 0.  Similar to machine translation, it is becoming relatively standard for a major TMS.  Keeps the code simpler.  Else we'll have to remove it later.